src/components/LeaderList/LeaderList.scss

```diff
@@ -0,0 +1,66 @@
+.leaderList {
+  width: 80%;
+  //height: 1000px;
+  margin: 20px auto;
+  box-sizing: border-box;
+  padding: 0 48px;
+  background-color: white;
```

Comment:
Best practice: move all colors to separate `_variables.scss` if they appear more than 1 time.